apps/aehttp/src/aehttp_dispatch_ext.erl

```diff
@@ -45,11 +47,16 @@
 handle_request(OperationID, Req, Context) ->
     run(queue(OperationID),
         fun() ->
-                handle_request_(OperationID, Req, Context)
+                ?TC(handle_request_(OperationID, Req, Context), Req)
```

Comment:
Do we want to keep timer:tc in the production code?

Reply:
It might be worthwhile to instead making it a metric (e.g. histogram)

Reply:
btw, `timer:tc/1` doesn't catch exceptions like `timer:tc/3` does, so in that sense, it's much more transparent.